yellowbrick/regressor/residuals.py

```diff
@@ -113,7 +117,7 @@ class PredictionError(RegressionScoreVisualizer):
     """
 
     def __init__(self, model, ax=None, shared_limits=True,
-                 bestfit=True, identity=True, **kwargs):
+                 bestfit=True, identity=True, alpha=1.0, **kwargs):
```

Comment:
If possible, I'd like to keep the default `alpha=0.75` like we had originally; I believe the alpha in the residuals plots is important to be able to quickly identify clusters of similar/duplicate errors and predictions.

Reply:
Yep, no problem

Reply:
Did this change get made? Am I seeing an older version of the code?

Reply:
Oops, I thought I had, but have now!